snapcraft/plugins/nodejs.py

```diff
@@ -64,6 +63,9 @@ def schema(cls):
             },
             'default': [],
         }
+        schema['properties']['node-engine'] = {
+            'type': 'string'
```

Comment:
Originally I did havethe default, but I'm not sure it's exactly what you want because you're almost always going to have a default that is out of date, as such I thought it better to make the developer take full control of this, and no default...